.gitignore

```diff
@@ -340,3 +340,6 @@ build/
 # vscode
 .vscode/
 .factorypath
+
+sdk/**/sdk/
+sdk/**/out/
```

Comment:
You might want to update the `# Build results` in the top of this file instead.  Make `out` to be something like `**/out/**` 
We can also add a comment about why we need it.